generator/lib/src/entity_resolver.dart

```diff
@@ -54,6 +55,12 @@ class EntityResolver extends Builder {
     // read all suitable annotated properties
     bool hasIdProperty = false;
     for (var f in element.fields) {
+
+      if (_transientChecker.hasAnnotationOfExact(f)) {
+        log.warning("  skipping property '${f.name}' in entity '${element.name}', marked with the @Transient annotation.");
```

Comment:
Not sure if this should be a warning instead of just "info". It's a user-defined annotation, not something we decide automagically. Also log.info. would allow a more straightforward generator output log:

current version with `log.warning()`:
```
[INFO] objectbox_generator:resolver on test/entity.dart:entity TestEntity(0:0)
[INFO] objectbox_generator:resolver on test/entity.dart:  property id(0:0) type:6 flags:1
[INFO] objectbox_generator:resolver on test/entity.dart:  property tString(0:0) type:9 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tLong(0:0) type:6 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tDouble(0:0) type:8 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tBool(0:0) type:1 flags:0
[WARNING] objectbox_generator:resolver on test/entity.dart:
  skipping property 'ignore' in entity 'TestEntity', marked with the @Transient annotation.
[INFO] objectbox_generator:resolver on test/entity.dart:  property tByte(0:0) type:2 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tShort(0:0) type:3 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tChar(0:0) type:4 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tInt(0:0) type:5 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tFloat(0:0) type:7 flags:0
```

an alternative with `log.info`:
```
[INFO] objectbox_generator:resolver on test/entity.dart:entity TestEntity(0:0)
[INFO] objectbox_generator:resolver on test/entity.dart:  property id(0:0) type:6 flags:1
[INFO] objectbox_generator:resolver on test/entity.dart:  property tString(0:0) type:9 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tLong(0:0) type:6 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tDouble(0:0) type:8 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tBool(0:0) type:1 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  skipping property ignore (annotated with @Transient)
[INFO] objectbox_generator:resolver on test/entity.dart:  property tByte(0:0) type:2 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tShort(0:0) type:3 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tChar(0:0) type:4 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tInt(0:0) type:5 flags:0
[INFO] objectbox_generator:resolver on test/entity.dart:  property tFloat(0:0) type:7 flags:0
```

Reply:
```suggestion
        log.info("  skipping property ${f.name} (annotated with @Transient)");
```